test/integration/admin/api/account/proxy_configs_controller_test.rb

```diff
@@ -111,6 +111,24 @@ def setup
     assert_same_elements expected_proxy_config_ids, response_proxy_config_ids
   end
 
+  test '#index of latest verisons filtered by host is only found among those' do
+    services = FactoryBot.create_list(:simple_service, 2, :with_default_backend_api, account: provider)
+    services.each do |service|
+      hosts = %w[3scale.localhost example.com]
+      FactoryBot.create(:proxy_config, proxy: service.proxy, environment: ProxyConfig::ENVIRONMENTS.first, content: ( { proxy: { hosts: hosts } }.to_json ) )
+    end
+
+    FactoryBot.create(:proxy_config, proxy: services.last.proxy, environment: ProxyConfig::ENVIRONMENTS.first, content: ( { proxy: { hosts: ['3scale.lvh.me'] } }.to_json ) )
+
+    get admin_api_account_proxy_configs_path(
+      environment: ProxyConfig::ENVIRONMENTS.first,
+      access_token: access_token_value(user: provider.admin_user)
+    ), params: {host: 'example.org', version: 'latest'}
+
+    expected_proxy_config_ids = services.first.proxy.proxy_configs.by_host('example.org').select(:id).map(&:id)
```

Comment:
`services.first` means hat it's neither of the two services created at the beginning of the test, right? Should we maybe add a second request with a different host and assert a different service is returned?

Reply:
not what it means 🙈 I will write it clearer following Josemi's advice 😉